tools/ovc/openvino/tools/ovc/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -250,6 +254,8 @@ def to_torch_tensor(tensor):
         return tuple(to_torch_tensor(x) for x in tensor)
     if isinstance(tensor, dict) and all(isinstance(k, str) for k in tensor.keys()):
         return dict((k, to_torch_tensor(x)) for k, x in tensor.items())
+    if tensor is None:
```

Comment:
```suggestion
    elif tensor is None:
```
and above also needs this slight correction